views/mc-answer.pug

```diff
@@ -0,0 +1,24 @@
+.form-group
+	p
+		| Is Correct: 
```

Comment:
can we use a loop here rather than duplicating code?

Reply:
yes we can!